src/modules/Builder/components/ConditionRow/ConditionRow.utils.tsx

```diff
@@ -58,26 +58,52 @@ const scoreItemTypes = [
   ItemResponseType.SingleSelection,
   ItemResponseType.MultipleSelection,
 ];
-const itemFlowItemTypes = [
-  ...scoreItemTypes,
-  ItemResponseType.Date,
-  ItemResponseType.NumberSelection,
-  ItemResponseType.Time,
-  ItemResponseType.TimeRange,
-  ItemResponseType.SingleSelectionPerRow,
-  ItemResponseType.MultipleSelectionPerRow,
-  ItemResponseType.SliderRows,
+
+const itemFlowItemGroups = [
+  [...scoreItemTypes, ItemResponseType.NumberSelection], // Group 1
+  [ItemResponseType.Date, ItemResponseType.Time, ItemResponseType.TimeRange], // Group 2
+  [
+    ItemResponseType.SingleSelectionPerRow,
+    ItemResponseType.MultipleSelectionPerRow,
+    ItemResponseType.SliderRows,
+  ], // Group 3
```

Comment:
I believe the groups are understandable, so is the comments necessary ?
if you think the comments is helping to understand, I strongly believe that create other well named variable to represent those groups could be good.